packages/tupaia-web/src/features/ExportSettings/ExportDescriptionInput.tsx

```diff
@@ -3,33 +3,40 @@
  * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
-import { TextField } from '@material-ui/core';
 import { useExportSettings } from './ExportSettingsContext';
 import { ExportSettingLabel } from './ExportSettingLabel';
+import { TextField, OutlinedTextFieldProps } from '@material-ui/core';
+
+const EXPORT_DESCRIPTION_LIMIT = 250;
 
 const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
 `;
 
-const ExportDescriptionInputArea = styled(TextField)`
+const ExportDescriptionInputArea = styled((props: OutlinedTextFieldProps) => (
+  <TextField {...props} />
```

Comment:
Out of interest, is there any reason you need to do this instead of `styled(TextField)`?

Reply:
Oh yes, because it was defaulting to `OutlinedTextFieldProps` which wasn't allowing me to configure `variant='outline'`